token.json

```diff
@@ -0,0 +1 @@
+{"token": "ya29.a0AeTM1idtetn_pW3fI_6snX_cf2uw2fjsu03eeuT7tjWysBkUeYThUErHXFC9JdGiL9pimloQs9I3xBQeZKnlWsQ7VcHISM9z7A0hPeIzg7z-2vrYIGLYhGSfj3lPukk_b2T6Q9Mf2kmaFw6B7OAJdx5nIJbSaCgYKAcASARESFQHWtWOmblXwWWFlYrPyid0vkY72Tw0163", "refresh_token": "1//06RF9T10KlaOCCgYIARAAGAYSNwF-L9IrYZr2-9geMfDxRm3Wzy1lxSQKSlzuFh-XLUazYrn3vuI4SMzr__1250UY9SVJta_dkQQ", "token_uri": "https://oauth2.googleapis.com/token", "client_id": "967750785532-e08f0o15hadakh8rvufda6rbrueftlht.apps.googleusercontent.com", "client_secret": "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", "scopes": ["https://www.googleapis.com/auth/calendar"], "expiry": "2022-12-13T21:57:40.737432Z"}
```

Comment:
Nbd for this project obvs, but weird that this got committed or maybe it's worth it to just get the `.gitignore` update in? And I think when I've encountered this, it's best to go back and edit the commit to remove, so no history is stored as well.